src/test/java/org/fundacionjala/coding/manuel/CesarCryptoTest.java

```diff
@@ -0,0 +1,109 @@
+package org.fundacionjala.coding.manuel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class to test CesarCrypto.
+ */
+public class CesarCryptoTest {
+
+    /**
+     * Object of the class to test.
+     */
+    private CesarCrypto cesarCrypto;
+
+    /**
+     * Method to init object.
+     */
+    @Before
+    public void init() {
+        cesarCrypto = new CesarCrypto();
+    }
+
+    /**
+     * Test encode method.
+     */
+    @Test
+    public void testEncode() {
+        final int firstKey = 3;
+        final int secondKey = 5;
+        assertEquals("KROD", cesarCrypto.encode("HOLA", firstKey));
+        assertEquals("ETWWT", cesarCrypto.encode("ZORRO", secondKey));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "ZORRO" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)